tests/sv/person_HassGetState.yaml

```diff
@@ -0,0 +1,65 @@
+language: sv
+tests:
+  - sentences:
+      - "var är Daniel"
+    intent:
+      name: HassGetState
+      slots:
+        domain: person
+        name: Daniel
+    response: "Daniel är hemma"
```

Comment:
Not sure where this person is defined, there is no fixture commitet to the test cases. Have you test run the commit before?